test/hummingbot/connector/exchange/ascend_ex/test_ascend_ex_order_book_tracker.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+import aiohttp
+
+from unittest import TestCase
+from unittest.mock import AsyncMock, patch
+
+from hummingbot.connector.exchange.ascend_ex.ascend_ex_api_order_book_data_source import \
+    AscendExAPIOrderBookDataSource
+from hummingbot.connector.exchange.ascend_ex.ascend_ex_order_book_tracker import AscendExOrderBookTracker
+from hummingbot.connector.exchange.ascend_ex import ascend_ex_constants as CONSTANTS
+from hummingbot.core.api_throttler.async_throttler import AsyncThrottler
+from test.hummingbot.connector.network_mocking_assistant import NetworkMockingAssistant
+
+
+class AscendExOrderBookTrackerTests(TestCase):
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.mocking_assistant = NetworkMockingAssistant()
+        self.session = None
+        self.throttler = AsyncThrottler(CONSTANTS.RATE_LIMITS)
+        self.tracker = AscendExOrderBookTracker(shared_client=self.session, throttler=self.throttler, trading_pairs=["BTC-USDT"])
+        self.mock_data_source = AsyncMock()
+        self.tracker._data_source = self.mock_data_source
+
+    def tearDown(self) -> None:
+        AscendExAPIOrderBookDataSource._trading_pair_symbol_map = {}
+        super().tearDown()
+
+    @patch("aiohttp.client.ClientSession.ws_connect", new_callable=AsyncMock)
+    async def do_listen_stream_trade(self, ws_connect_mock):
+        ret = {}
+
+        self.tracker._shared_client = aiohttp.ClientSession()
```

Comment:
It should not be required to initialize the session from outside the tracker (we can either provide it as part of the instance creation or let the tracker create it)